recipes/chunkio/all/patches/1.1.6-0003-disable-pic.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ebf3c4c..915fb28 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -7,7 +7,7 @@ set(CIO_VERSION_PATCH  6)
+ set(CIO_VERSION_STR "${CIO_VERSION_MAJOR}.${CIO_VERSION_MINOR}.${CIO_VERSION_PATCH}")
+ 
+ # CFLAGS
+-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
++#set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
It must be kept if shared because there is a vendored libs always build a static lib, so when chunkio is shared, it fails if PIC is not enabled.